pkg/common/k8s_utils.go

```diff
@@ -17,13 +17,17 @@ limitations under the License.
 package common
 
 import (
+	"encoding/json"
 	"fmt"
+	"sort"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
-	DeleteTagAnnotation = "kuadrant.io//delete"
+	DeleteTagAnnotation      = "kuadrant.io//delete"
```

Comment:
Was that "//" intended, or is it a typo?

Reply:
fixed